queries.yaml

```diff
@@ -21,9 +21,14 @@ sources:
 
   planet_osm_polygon:
     - template: planet_osm_polygon.jinja2
+      # landuse and pois start at 4
+      start_zoom: 4
```

Comment:
Like you mentioned, this just matches what the behavior was previously. The `landuse` and `pois` layers queries started at zoom 4. If we go to 3 and we have some records that have a min zoom < 4, then we will start showing those features at the lower zooms. Is this what we want? If so, perhaps we should do it in a follow up issue/pr just to make that clearer?

Btw, I'm running a query on the dev database just to see what the smallest min zoom values are for pois and landuse in the polygon table. I'll report back when the results come through.